src/registry.js

```diff
@@ -1,16 +1,12 @@
 var Registry = {
-  registry: [],
-  register: function(entity) {
-    this.registry.push(entity);
-  },
-
-  entityFor: function(name) {
-    for (var i = 0; i < this.registry.length; i++) {
-      var entity = this.registry[i];
-
-      if (entity.handles(name)) {
-        return entity;
-      }
+    registry: [],
+    register: entity => this.registry.push(entity),
```

Comment:
This is giving me an error 'Cannot read property 'push' of undefined'

Reply:
I changed this back to a standard function for now